### PR TITLE
demos: improved UX for sidebar

### PR DESCRIPTION
**Pull request by benouat**
Instead of a long description, I'll put a GIF

Kapture 2020-08-07 at 10 27 25

### CODE REVIEW

**Comment by benouat** [comment]
I did the change from `*ngIf` to `[ngbCollapse]` as suggested by @jnizet 

Maybe we should also introduce **Guides** as a top level navigation item in the blue header ?

@pkozlowski-opensource @maxokorokov are you guys ok with the update behind the PR ? 🤔 visually speaking I mean

**Comment by pkozlowski-opensource** [comment]
@benouat is it about grouping guides into one block / section? If so, I think this is a good move 👍

**Comment by benouat** [comment]
> is it about grouping guides into one block / section? If so, I think this is a good move 👍
 
yes, part of it is that, other part is to collapse the non-active section, or expand the active one 😄

**Comment by pkozlowski-opensource** [comment]
> the non-active section, or expand the active one

Hmm, not sure about collapsing the components section - maybe we want to see a list of components all the time? Just a random thought, not feeling strongly either way.

**Comment by benouat** [comment]
it's actually mimicking bootstrap's website. You see the list of components while you are on a component's page. Same for guides, you see the list while you are on a guide...

File: demo/src/app/app.routing.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ const routes: Routes = [
   {path: 'getting-started', component: GettingStartedPage},
   {path: 'animations', component: AnimationsPage},
   {path: 'positioning', component: PositioningPage},
-  {path: 'components', pathMatch: 'full', redirectTo: 'components/alert'},
+  {path: 'components', pathMatch: 'full', redirectTo: 'components/accordion'},
```
Comment (maxokorokov) [inline demo/src/app/app.routing.ts:34]: 🎉

**Review by maxokorokov — APPROVED** [review]
@benouat, I like the idea

- I think I would extract i18n from the getting started to the "Guides"
- Also "Guides" is not easily discoverable now, I would either just put them in the header or put the "Getting Started" under the guides menu:

```
Guides
  Getting Started
  Animations
  I18n
  Positioning

Components
```

**Comment by benouat** [comment]
I kept `Getting Started` as stand alone section, and introduced `Guides` in the top level navigation.

image

I also moved `I18n` out of `Getting Started` to move it to `Guides`
I changed  a bit the spacing and added a background color when hovered on each nav item.

Here is the new demo
Kapture 2020-08-25 at 15 54 41

**Comment by benouat** [comment]
I also moved guides to a new `#/guides/` route but kept redirection for both old `/animations` and `/positioning` routes

**Pull request merged**